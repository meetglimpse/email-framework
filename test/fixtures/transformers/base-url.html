<html>
  <head>
    <style>
      .test {
        background-image: url('image.jpg');
        background: url('image.jpg');
        background-image: url('https://preserve.me/image.jpg');
        background: url('https://preserve.me/image.jpg');
      }

      .test-2 {
        background-image: url("image.jpg");
        background: url("image.jpg");
        background-image: url("https://preserve.me/image.jpg");
        background: url("https://preserve.me/image.jpg");
      }

      .test-3 {
        background-image: url(image.jpg);
        background: url(image.jpg);
        background-image: url(https://preserve.me/image.jpg);
        background: url(https://preserve.me/image.jpg);
      }
    </style>
  </head>

  <body>
    <img src="test.jpg">
    <img src="https://example.com/test.jpg">

    <img src="image1.jpg" srcset="image1-HD.jpg 2x,image1-phone.jpg 100w">

    <img src="https://example.com/image2.jpg" srcset="https://example.com/image2-HD.jpg 2x, https://example.com/image2-phone.jpg 100w">

    <picture>
      <source media="(max-width: 799px)" srcset="elva-480w-close-portrait.jpg">
      <source media="(min-width: 800px)" srcset="elva-800w.jpg">
      <img src="elva-800w.jpg" alt="...">
    </picture>

    <video width="250" poster="flower.jpg">
      <source src="media/flower.webm" type="video/webm">
      <source src="https://example.tv/media/flower.mp4" type="video/mp4">
      <track default kind="captions" srclang="en" src="media/tracks/friday.vtt">
    </video>

    <audio src="media/sample.mp3">
      Fallback content
    </audio>

    <embed type="video/webm" src="media/flower.mp4" width="250" height="200">

    <iframe width="300" height="200" src="embed.html"></iframe>

    <input type="image" src="image.jpg" alt="">

    <script src="javascript.js"></script>

    <div>
      <!--[if mso]>
      <v:image xmlns:v="urn:schemas-microsoft-com:vml" src="image.jpg" style="width:600px;height:400px;" />
      <v:rect fill="false" stroke="false" style="position:absolute;width:600px;height:400px;">
      <v:textbox inset="0,0,0,0"><div><![endif]-->
        <div>test</div>
      <!--[if mso]></div></v:textbox></v:rect><![endif]-->
    </div>

    <!--[if mso]>
      <v:image src="image-2.jpg" xmlns:v="urn:schemas-microsoft-com:vml" style="width:600px;height:400px;" />
    <![endif]-->

    <!--[if mso]>
      <v:image xmlns:v="urn:schemas-microsoft-com:vml" src="https://example.com/image-3.jpg" style="width:600px;height:400px;" />
    <![endif]-->

    <table>
      <tr>
        <td background="image.png" bgcolor="#7bceeb" width="120" height="92" valign="top">
          <!--[if gte mso 9]>
          <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:120px;height:92px;">
            <v:fill type="tile" src="image.png" color="#7bceeb" />
            <v:textbox inset="0,0,0,0">
          <![endif]-->
          <div>test</div>
          <!--[if gte mso 9]>
            </v:textbox>
          </v:rect>
          <![endif]-->
        </td>
      </tr>
    </table>

    <!--[if mso]>
      <v:fill type="tile" src="image.png" color="#7bceeb" />
    <![endif]-->

    <!--[if mso]>
      <v:fill type="tile" src="https://example.com/image.png" color="#7bceeb" />
    <![endif]-->
  </body>
</html>
