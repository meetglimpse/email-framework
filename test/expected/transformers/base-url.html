<html>
  <head>
    <style>.test {
        background-image: url('https://example.com/image.jpg');
        background: url('https://example.com/image.jpg');
        background-image: url('https://preserve.me/image.jpg');
        background: url('https://preserve.me/image.jpg');
      }

      .test-2 {
        background-image: url("https://example.com/image.jpg");
        background: url("https://example.com/image.jpg");
        background-image: url("https://preserve.me/image.jpg");
        background: url("https://preserve.me/image.jpg");
      }

      .test-3 {
        background-image: url(https://example.com/image.jpg);
        background: url(https://example.com/image.jpg);
        background-image: url(https://preserve.me/image.jpg);
        background: url(https://preserve.me/image.jpg);
      }</style>
  </head>

  <body>
    <img src="https://example.com/test.jpg">
    <img src="https://example.com/test.jpg">

    <img src="https://example.com/image1.jpg" srcset="https://example.com/image1-HD.jpg 2x, https://example.com/image1-phone.jpg 100w">

    <img src="https://example.com/image2.jpg" srcset="https://example.com/image2-HD.jpg 2x, https://example.com/image2-phone.jpg 100w">

    <picture>
      <source media="(max-width: 799px)" srcset="https://example.com/elva-480w-close-portrait.jpg">
      <source media="(min-width: 800px)" srcset="https://example.com/elva-800w.jpg">
      <img src="https://example.com/elva-800w.jpg" alt="...">
    </picture>

    <video width="250" poster="https://example.com/flower.jpg">
      <source src="https://example.com/media/flower.webm" type="video/webm">
      <source src="https://example.tv/media/flower.mp4" type="video/mp4">
      <track default="" kind="captions" srclang="en" src="https://example.com/media/tracks/friday.vtt">
    </video>

    <audio src="https://example.com/media/sample.mp3">
      Fallback content
    </audio>

    <embed type="video/webm" src="https://example.com/media/flower.mp4" width="250" height="200">

    <iframe width="300" height="200" src="https://example.com/embed.html"></iframe>

    <input type="image" src="https://example.com/image.jpg" alt="">

    <script src="https://example.com/javascript.js"></script>

    <div>
      <!--[if mso]>
      <v:image xmlns:v="urn:schemas-microsoft-com:vml" src="https://example.com/image.jpg" style="width:600px;height:400px;" />
      <v:rect fill="false" stroke="false" style="position:absolute;width:600px;height:400px;">
      <v:textbox inset="0,0,0,0"><div><![endif]-->
        <div>test</div>
      <!--[if mso]></div></v:textbox></v:rect><![endif]-->
    </div>

    <!--[if mso]>
      <v:image src="https://example.com/image-2.jpg" xmlns:v="urn:schemas-microsoft-com:vml" style="width:600px;height:400px;" />
    <![endif]-->

    <!--[if mso]>
      <v:image xmlns:v="urn:schemas-microsoft-com:vml" src="https://example.com/image-3.jpg" style="width:600px;height:400px;" />
    <![endif]-->

    <table>
      <tr>
        <td background="https://example.com/image.png" bgcolor="#7bceeb" width="120" height="92" valign="top">
          <!--[if gte mso 9]>
          <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:120px;height:92px;">
            <v:fill type="tile" src="https://example.com/image.png" color="#7bceeb" />
            <v:textbox inset="0,0,0,0">
          <![endif]-->
          <div>test</div>
          <!--[if gte mso 9]>
            </v:textbox>
          </v:rect>
          <![endif]-->
        </td>
      </tr>
    </table>

    <!--[if mso]>
      <v:fill type="tile" src="https://example.com/image.png" color="#7bceeb" />
    <![endif]-->

    <!--[if mso]>
      <v:fill type="tile" src="https://example.com/image.png" color="#7bceeb" />
    <![endif]-->
  </body>
</html>
